### ISSUE TITLE
Alamopy gaussian rbf basis function evaluate surrogate issue

### ISSUE THREAD

**Issue by Clint321836** [IDAES/idaes-pse#1223]
I'm getting a syntax error in the alamopy file at line 1224 when trying to create the lambda for the surrogate model. Looking through the traceback, it looks like the lambda is attempting to read a string from alamo and convert it to python syntax. 

It works fine for all of the other basis functions I've tried, but for the grbf Alamo returns surrogate expressions like this:

'Y1 ==  - 482267.41297314106486738 * exp ( -1. * sum of distances from initial data point 32 )'

It looks like the function has no idea what to do with the sum of distances from initial data point 32 variable and needs a way to parse these and generate the lambda.

Here's a simpler sample code that produces the same issue of not being able to parse the alamo output:
[alamopy_grbf_error.txt](https://github.com/IDAES/idaes-pse/files/12052334/alamopy_grbf_error.txt)

**Comment by andrewlee94** [comment]
@Clint321836 Yes, the existing lambda function would not know how to handle that, and off hand I do not know of an easy way to do that as the current ALAMOpy interface has no idea about distances from initial points. We also do not have a lot of time available to looking into supporting this within the IDAES team right now. Unless you have the time and can think of a way to support these, then I think that for now ALAMOpy will not support radial basis functions and that configuration option should be removed.

For radial basis functions, I would suggest looking at Pysmo instead, as it already supports these: https://idaes-pse.readthedocs.io/en/stable/explanations/modeling_extensions/surrogate/api/pysmo/index.html?highlight=pysmo
